django/cantusdb_project/requirements.txt

```diff
@@ -13,6 +13,7 @@ Django==4.1.7
 django-autocomplete-light==3.5.1
 django-extra-views==0.13.0
 django-quill-editor==0.1.40
+django_debug_toolbar==3.8.1
```

Comment:
I think this is fine...

Reply:
> changed some html formatting

i.e. mostly splitting lines down into nested tags on different lines?

Reply:
exactly.